dwave/optimization/_model.pyx

```diff
@@ -93,17 +93,39 @@ cdef class _Graph:
         return value
 
     def decision_state_size(self):
-        r"""An estimated size, in bytes, of the model's decision states.
+        """Return an estimate of the size, in bytes, of a model's decision states.
+
+        For more details, see :meth:`.state_size()`.
+        This method differs by only counting the state of the decision variables.
```

Comment:
```suggestion
        This method differs by counting the state of only the decision variables.
```

The rational here is that when you read "This method differs by only" you expect the diff ("not counting non-decision variables" but you get what the updated method does.